ambari-server/src/main/resources/stacks/HDP/2.6/upgrades/config-upgrade.xml

```diff
@@ -51,6 +51,23 @@
           </definition>
         </changes>
       </component>
+      <component name="DATANODE">
```

Comment:
This should be in the HDP 3.0 stack since the feature to turn this on is expected for HDP 3.0, not HDP 2.6.

CC: @smolnar82

Reply:
I am dropping this concern since misunderstood the usage of the config-upgrade.xml file.

Reply:
added missing config change references to 2.6->3.0 upgrade pack